src/FormBody.php

```diff
@@ -9,7 +9,7 @@
 use Psr\Http\Message\StreamInterface;
 
 
-Final class FormBody {
+final class FormBody {
```

Comment:
specificity over caps... This really should come with a coding style issue and PR first

Reply:
It is the same as the previous comment.